app/views/lessons/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="container">
+    <h1><%= @lesson.category.title %></h1>
+    <table class="table table-striped table-dark">
+        <tbody>
+            <tr>
+                <th>Word</th>
+                <th>Your Answer</th>
+                <th>Correct Answer</th>
+            </tr>
+            <% @lesson.answers.each do |a|%>
+                <tr>
+                    <td><%= a.word.content %></td>
+                    <td><%= a.choice.content %></td>
+                    <td><%= a.choice.word.choices.where(correct_answer: true).first.content %></td>
+                    <td><% if a.choice.content == a.choice.word.choices.where(correct_answer: true).first.content %>
```

Comment:
Please change "a.choice.word.choices.where(correct_answer: true).first.content" to  "a.choice.word.choices.find_by(correct_answer: true).content".